src/Symfony/Component/Security/Core/Tests/Exception/UsernameNotFoundExceptionTest.php

```diff
@@ -1,26 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Security\Core\Tests\Exception;
-
-use PHPUnit\Framework\TestCase;
-use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
-
-class UsernameNotFoundExceptionTest extends TestCase
-{
-    public function testGetMessageData()
```

Comment:
Better mark it as `@group legacy` until 6.0

Reply:
I've moved the legacy test in the `UserNotFoundExceptionTest`.